doc/manual/source/threat-model.rst

```diff
@@ -0,0 +1,72 @@
+
+The data Routinator retrieves from the repositories, as well as the repositories themselves, are untrusted. Objects are validated using cryptographic signatures that can only be made by resource holders, and contents are checked according to the schema, and data is checked against the constraints in the certificate.
+
+.. image:: img/routinator-data-flow.png
+
+There are two kinds of data:
+
+* **Repository data** is the data from the public repositories to Routinator. This is unvalidated and still untrusted.
+* Validated **payload data** is the data Routinator parsed from the repositories and is sent to the routers. This data is validated and should be trustworthy.
+
+
+Assumptions
+-----------
+
+Routinator should:
+
+* Retrieve repository data from repositories in a reasonable time
+* Parse this repository data correctly and in a reasonable time
+* Not let one repository impact the outcome of another repository, unless that repository is a child of the former repository
+* For data thus retrieved, be able to provide correct, complete, non-malformed, validated payloads to routers:
+
+  * Run correctly even if the input data is malformed
+  * Only contain validated and correct data in the data sent to routers
+  * Reject invalid data without impacting availability and integrity of valid data
+
+* Keep resource (memory, file system, etc.) consumption within reasonable bounds
+
+General assumptions:
+
+* The user uses an authentic copy of Routinator
+* An attacker does not have administrative control over the host system Routinator runs on
+* An attacker does not have write-access to Routinator’s cache directories
+* The user uses trusted TALs, and the TALs enabled by default are trusted
+* Advances in cryptoanalytic attacks against the cryptographic primitives used by the RPKI have not occurred
+* Published repository data is authentic, i.e., resource holders maintain confidentiality of the key pairs used to sign statements in the RPKI
+
+Routinator guarantees the following: 
+
+* Unsigned or incorrectly signed objects in the RPKI will not end up in the validated RPKI payloads provided to routers
+* For a repository that is reachable, when repository data set up in a reasonable and best practice compliant manner, modifications in the RPKI will be processed by Routinator in a reasonable time.
+
+An adversary with network access on-path between Routinator and a repository could:
+
+* Block access to the repository or degrade throughput of access to repository data
+* Downgrade the transport used for repository access from RRDP (which provides confidentiality and integrity at the transport level) to rsync (which does not). This behaviour was decided by the community. After such a downgrade, the attacker can: 
+
+  * Inject objects, withhold objects and manipulate objects, leading to rejection of otherwise valid repository data.
```

Comment:
```suggestion
  * Inject objects, withhold objects and manipulate objects at the transport level, leading to rejection by Routinator of otherwise valid repository data
```